libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/feedback/FeedbackFlowListener.kt

```diff
@@ -23,4 +23,9 @@ interface FeedbackFlowListener {
      * @param arrivalFeedbackItem the overall experience of a navigation
      */
     fun onArrivalExperienceFeedbackFinished(arrivalFeedbackItem: FeedbackItem)
+
+    /**
+     * The callback gets triggered when **Feedback flow** finished.
```

Comment:
```suggestion
     * The callback gets triggered when the view is dismissed.
```

Reply:
it's not correct, count of views mights from 1 to 2. Actually **Feedback Flow** is a good name because we register `FeedbackFlowListener` so we notify when the flow finish.